enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/server/CoreServerModule.java

```diff
@@ -136,29 +132,6 @@ public CoreServerModule( IdentityModule identityModule, final PlatformModule pla
 
         LifeSupport servicesToStopOnStoreCopy = new LifeSupport();
 
-        if ( config.get( OnlineBackupSettings.online_backup_enabled ) )
-        {
-            platformModule.dataSourceManager.addListener( new DataSourceManager.Listener()
-            {
-                @Override
-                public void registered( NeoStoreDataSource dataSource )
-                {
-                    servicesToStopOnStoreCopy.add( pickBackupExtension( dataSource ) );
-                }
-
-                @Override
-                public void unregistered( NeoStoreDataSource dataSource )
-                {
-                    servicesToStopOnStoreCopy.remove( pickBackupExtension( dataSource ) );
-                }
-
-                private OnlineBackupKernelExtension pickBackupExtension( NeoStoreDataSource dataSource )
-                {
-                    return dataSource.getDependencyResolver().resolveDependency( OnlineBackupKernelExtension.class );
-                }
-            } );
-        }
-
```

Comment:
Is it always enabled, and the extension is registered automatically?

Reply:
It should always be disabled for CC (there is no need in maintaining it, since the neo4j-admin will be the same version as the cc running). But you are saying it is registered automatically? That is what I need to avoid (`@Ignore` in OnlineBackupCommandCcIt)

Reply:
Resolved - CC (core+RR) no longer have the backup protocol